src/va_list.cr

```diff
@@ -18,7 +18,15 @@ struct VaList
     end
   end
 
-  {% if compare_versions(Crystal::VERSION, "0.33.0-0") > 0 %}
+  {% if flag?(:aarch64) %}
+    # Do not call this, use C wrappers calling the va_arg macro for the types you need.
+    #
+    # Clang implements va_arg on AArch64 like this: https://github.com/llvm/llvm-project/blob/a574edbba2b24fcfb733aa2d82308131f5b7d2d6/clang/lib/CodeGen/TargetInfo.cpp#L5677-L5921
+    # If somebody wants to fix the va_arg instruction on AArch64 upstream, or port the above here, that would be welcome
+    def next(type)
+      \{% raise "The LLVM va_arg instruction is broken on AArch64, please implement wrappers in C calling the va_arg macro for the types you need and bind to them" %}
```

Comment:
I really don't like how this message reads. Even as a knowledgeable user it's just hard to get the point.

Particularly (as I assume this is a *user*-facing message):

* I don't immediately care that it's LLVM's fault.
* I don't immediately want to see what the workaround is
  (also takes a while to realize that this *is* just a suggestion of a workaround).

I want to foremost plainly see what is not working.

But unfortunately, the entirety of the message is about the prior two irrelevant points.

Reply:
Suggestions for a better message rather than just pointing out what you don't like here?